spec/schema.json

```diff
@@ -320,6 +448,70 @@
         }
       }
     },
+    "GeneralPartnerData": {
+       "type": "object",
```

Comment:
Is a field for the 'statement' missing? The "I confirm that..." checkbox.

Reply:
Added